sktime/param_est/seasonality.py

```diff
@@ -407,3 +405,71 @@ def get_test_params(cls, parameter_set="default"):
         params2 = {"candidate_sp": [3, 7, 12]}
 
         return [params1, params2]
+
+
+class Seasonality(BaseParamFitter):
```

Comment:
I wouldn´t call it `Seasonality`, maybe be more specific, e.g., `SeasonalityPeriodogram` or similar

Reply:
Done!

Reply:
thanks